tests/ecosystem/upgrade/test_configuration.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+import pytest
+from ocs_ci.framework.pytest_customization.marks import (
+    order_pre_upgrade, order_post_upgrade
+)
+from ocs_ci.ocs.resources import pod
+
+log = logging.getLogger(__name__)
+
+
+def get_crush_map():
+    """
+    Get decompiled CRUSH map from ceph toolbox pod.
+
+    Returns:
+        str: Multiline string representing current Ceph CRUSH map
+    """
+    ct_pod = pod.get_ceph_tools_pod()
+    file_comp = '/tmp/crush_comp'
+    file_decomp = '/tmp/crush_decomp'
+    ct_pod.exec_ceph_cmd(f"ceph osd getcrushmap -o {file_comp}")
+    ct_pod.exec_ceph_cmd(f"crushtool -d {file_comp} -o {file_decomp}")
+    return ct_pod.exec_bash_cmd_on_pod(f"cat {file_decomp}")
+
+
+@pytest.fixture(scope='session')
+def pre_upgrade_crush_map():
+    """
+    Loads CRUSH map before upgrade by `test_load_crush_map` test case.
+
+    Returns:
+        str: String consisting of CRUSH map before upgrade
+    """
+    crash_map = get_crush_map()
+    log.info(f"Pre upgrade CRASH map: {crash_map}")
+    return crash_map
+
+
```

Comment:
Polarion ID?

Reply:
Please change CRASH to CRUSH wherever applicable

Reply:
done